hack/install-pgo.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sudo git clone https://github.com/CrunchyData/postgres-operator-examples.git
+
+kubectl apply -k postgres-operator-examples/kustomize/install/namespace
```

Comment:
do we need two different installs ? think just the default install ?

Reply:
No, I'll remove this.